modules/xfeatures2d/include/opencv2/xfeatures2d.hpp

```diff
@@ -145,6 +145,21 @@ class CV_EXPORTS LUCID : public DescriptorExtractor
 };
 
 
+/*
+* LATCH Descriptor
+*/
+
+/** latch Class for computing the LATCH descriptor.
+If you find this code useful, please add a reference to the following paper in your work:
+Gil Levi and Tal Hassner, "LATCH: Learned Arrangements of Three Patch Codes", arXiv preprint arXiv:1501.03719, 15 Jan. 2015
```

Comment:
can you also add some descriptions for parameters and the algorithm itself. E.g. that this is binary descriptor extractor, that it's recommended to use it with FAST or AGAST or whatever